### DISCUSSION TITLE
pH7Builder v18.0 Beta 1

### DISCUSSION THREAD

**Discussion by pH-7** [pH7Software/pH7-Social-Dating-CMS#1079] [Release Announcements]
- First version that requires at least PHP 8.0 or newer (previously, it was at least PHP 7.4 since v16.5.0)
- Several improvements, enhancements and bug fixes

Feel free to test it and [open an issue](https://github.com/pH7Software/pH7-Social-Dating-CMS/issues) if you find bugs or something weird in order to provide quality assurance for the next stable version 💫

Enjoy 🎉 Hope you will ❤️ LOVE it 😸

<hr /><em>This discussion was created from the release <a href='https://github.com/pH7Software/pH7-Social-Dating-CMS/releases/tag/18.0.0-beta.1'>pH7Builder v18.0 Beta 1</a>.</em>

**Comment by edvorg** [comment]
Yes attach pics as images to this email
